src/adb.js

```diff
@@ -1 +1,353 @@
-// https://github.com/appcelerator/titanium_mobile/blob/master/node_modules/titanium-sdk/lib/adb.js
+import 'babel-polyfill';
+import 'source-map-support/register';
+import fs from 'fs';
+import path from 'path';
+import wrench from 'wrench';
+import { spawn } from 'child_process';
+
+import { Connection } from './connection';
+import { Emulator, EmulatorManager } from './emulator';
+import * as util from './util';
+import * as sdk from './sdk';
+
+/**
+ * Debug flag that is enabled via the android.debugadb setting.
+ */
+let DEBUG = false;
+
+/**
+ * Creates an ADB object.
+ * @class
+ * @classdesc Provides methods to interact with the Android Debug Bridge (ADB).
+ * @constructor
+ */
+export class ADB {
+	constructor(config) {
+		this.config = config;
+		if (config && config.get('android.debugadb', false)) {
+			DEBUG = true;
+		}
+	}
+
+	/**
+	 * Returns the version of the ADB server.
+	 * @returns {Promise}
+	 */
+	version() {
+		return new Connection(this)
+			.exec('host:version')
+			.then(data => {
+				return '1.0.' + parseInt(data, 16);
+			})
+			.catch(err => {
+				console.log('adb version: ', err);
+			});
+	}
+
+	/**
+	 * Runs the specified command on the Android emulator/device. Note that ADB
+	 * converts all \n to \r\n. So data will probably be larger than the original
```

Comment:
Fix verbiage "So data...". Try "converts all '\n' to '\r\n' and thus data will probably...".